server/controllers/sitterControl.ts

```diff
@@ -1,21 +1,17 @@
 import express from "express";
 import Sitter from "../models/sitter";
 
-const find = () => {
```

Comment:
This lines of codes might been added by mistake trying to import a function declaration. I have removed it and import the function from the model instead.